### PR TITLE
Translate cookbook/index.md

### PR DESCRIPTION
**Pull request by migsalazar**
## Description of Problem
None. 

## Proposed Solution
Translate `src/cookbook/index.md`

## Additional Information
None.

### CODE REVIEW

File: src/cookbook/index.md [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
 :::
 
-## Cookbook Contributions
+## Contribuciones del libro de recetas
 
-### What we're looking for
+### Lo que estamos buscando
 
-The Cookbook gives developers examples to work off of that both cover common or interesting use cases, and also progressively explain more complex detail. Our goal is to move beyond a simple introductory example, and demonstrate concepts that are more widely applicable, as well as some caveats to the approach.
+El libro de recetas proporciona ejemplos típicos de la vida real o interesantes para que los desarrolladores puedan usar como modelo, y también explica progresivamente detalles más complejos. Nuestro objetivo es ir más allá de un simple ejemplo introductorio, y demostrar los conceptos que son más aplicables, así como algunas advertencias al enfoque utilizado.
```
Comment (Tintef) [inline src/cookbook/index.md]: What do you think about this?
```suggestion
El libro de recetas proporciona ejemplos típicos de la vida real o interesantes para que los desarrolladores puedan usar como modelo, y también explica progresivamente detalles más complejos. Nuestro objetivo es ir más allá de un simple ejemplo introductorio, y demostrar los conceptos que son ampliamente aplicables, así como algunas advertencias al enfoque utilizado.
```

File: src/cookbook/index.md [hunk 2]
```diff
@@ -1,77 +1,77 @@
...
-- Explain the process, rather than just the end result
-- Explain the pros and cons of your strategy, including when it is and isn't appropriate
-- Mention alternative solutions, if relevant, but leave in-depth explorations to a separate recipe
+- Resolver un problema específico y común.
+- Comenzar con el ejemplo más simple posible.
+- Introducir las complejidades una a la vez.
+- Enlazar a otros documentos, en lugar de re-explicar conceptos.
+- Describir el problema, en lugar de asumir familiaridad.
+- Explicar el proceso, en lugar de solo mostrar el resultado.
+- Explicar las ventajas y desventajas de su estrategia, incluso, cuándo es apropiado y cuándo no.
```
Comment (Tintef) [inline src/cookbook/index.md]: Should it be `apropiada`? As you are talking about `la estrategia`? 🤔

**Review by Tintef — COMMENTED** [review]
Great work! 💪

File: src/cookbook/index.md (hunk 2 above)
Comment (migsalazar) [reply src/cookbook/index.md]: Yes! You're right!

File: src/cookbook/index.md (hunk 1 above)
Comment (migsalazar) [reply src/cookbook/index.md]: Agree!

**Comment by migsalazar** [comment]
Updated! Thank you @Tintef

**Review by Tintef — APPROVED** [review]

**Pull request merged**